transports/sql/transactions_native_sqltransport_[6,).partial.md

```diff
@@ -6,3 +6,6 @@ There are two available options within native transaction level:
 > This transaction is not shared outside of the message receiver. That means there is a possibility of persistent side-effects when processing fails, i.e. [ghost messages](/nservicebus/concepts/glossary.md#ghost-message) might occur.
 
  * **SendsAtomicWithReceive** - This mode is similar to the `ReceiveOnly`, but transaction is shared with sending operations. That means the message receive operation and any send or publish operations are committed atomically.
+
+> [!WARNING]
+> If using [SQL Server transport](/transports/sql/) and [SQL Persistence with the Microsoft SQL Server dialect](/persistence/sql/dialect-mssql.md) with different connection strings, without [outbox](/nservicebus/outbox/) enabled, in this transaction mode [the persistence uses the connection and transaction context established by the transport when accessing saga data](/persistence/sql/sqlserver-combining-persistence-wth-transport.md).
```

Comment:
This does not read well, needs improvement.

Reply:
Maybe first step is to start the sentence with "When using ...."

Reply:
I have reworded this to make it simpler.